internal/dataplane/kongstate/plugin_test.go

```diff
@@ -767,6 +769,8 @@ func TestKongPluginFromK8SPlugin(t *testing.T) {
 }
 
 func TestPlugin_SanitizedCopy(t *testing.T) {
+	// this needs a static random seed because some auths generate random values
+	uuid.SetRand(rand.New(rand.NewSource(1))) //nolint:gosec
```

Comment:
```suggestion
```
I believe this test doesn't need that.

Reply:
Yeah, leftover from when I tried to do it for every type of entity I think.